apps/native-component-list/screens/GoogleSignInScreen.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { GoogleSignIn, Localization } from 'expo';
+import { Image, StyleSheet, Text, View } from 'react-native';
+
+import GoogleSignInButton from '../components/GoogleSignInButton';
+
+export default class GoogleSignInScreen extends React.Component {
+  static navigationOptions = {
+    title: 'Google Sign-In',
+  };
+
+  state = { user: null };
+
+  componentDidMount() {
+    this._configureAsync();
+  }
+
+  _configureAsync = async () => {
+    const language = Localization ? Localization.locale : 'en-US';
+    try {
+      await GoogleSignIn.initAsync({
+        isOfflineEnabled: true,
+        isPromptEnabled: true,
+        clientId: '603386649315-vp4revvrcgrcjme51ebuhbkbspl048l9.apps.googleusercontent.com',
```

Comment:
Is it going to work for all Expo developers (for example for Facebook apps in development mode one would need to invite all other developers testing NCL for them to be able to authenticate with Facebook to an unpublished Facebook app)? Certainly it could be extracted to some constant at the top of the file. 🙂

Reply:
AFAIK this module will only work in standalone. You need a URI scheme defined in your info.plist to run. This is really just for testing. The client ID value here is an override value, the real value will be inferred from a `GoogleServices-Info.plist`. I think the client ID corresponds to the custom URI scheme, so this probably only works for this individual value.

I imagine that in order to get the firebase modules working, we will need a way to include a custom `GoogleServices-Info.plist` before building a standalone IPA. Once we add this method, I may just remove the `clientId` value altogether. 

> Re: Facebook

this works for everyone